src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Microsoft.AspNetCore.Diagnostics.Middleware.csproj

```diff
@@ -8,6 +8,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
+    <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
```

Comment:
NIT: Does it hurt to set this property to true when a project is not referencing the configuration binder package? If not, then we might want to consider defaulting this to true in directory.build.props to avoid having to add it in most places. And to that same line, should we consider adding the generator by default to all projects too? I'm assuming libraries not using configuration will simply no-op for this generator (which may include a slight perf regresion in build time) but this may be beneficial for new libraries in the future.

Reply:
(Totally my opinion, I'm curious to hear others')

Since this source generator is off by default, I thought it would be better to continue to keep it off by default. It isn't really "most places". It is 12 libraries. By comparison, `UseLoggingGenerator` is used in 13 libraries, and it isn't defaulted to be "on by default".